__tests__/__unit__/extension.unit.test.ts

```diff
@@ -874,18 +874,94 @@ describe("Extension Unit Tests", () => {
         expect(packageJsonContent.contributes.menus["view/item/context"]).toMatchSnapshot();
     });
 
-    it("Call Change File type", async () => {
-        const node = new ZoweUSSNode("node", vscode.TreeItemCollapsibleState.None, ussNode, null, null);
-        const response: zowe.IZosFilesResponse = {
-            success: true,
-            commandResponse: null,
-            apiResponse: {
-                etag: "132"
-            }
-        };
-        ussFile.mockResolvedValueOnce(response);
-        const res = ussActions.changeFileType(node, false, testUSSTree);
-        expect(res).not.toBeUndefined();
+    it("Testing that refreshPS correctly executes with and without error", async () => {
```

Comment:
Are these new tests or did we move them from somehow? I am a little confused because I though we are sliming down extension.unit.test.ts

Reply:
No, these haven't been refactored yet. So they are left here.

By the very end this file will definitely be cleaned up. I have already refactored the tests file (see [this branch](https://github.com/zowe/vscode-extension-for-zowe/tree/refactor-unit-tests-extension) but it should only be merged after all the other test files are refactored, so it can be updated to match all the changes and emptied out only once.

Reply:
The reason why I am asking is that I see it as `+` - additions to the file and that confuses me.

Reply:
I would just leave it there because we will go through the whole file again later anyway. So as long as things are being added, not removed, it should be fine.